RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -35,6 +36,7 @@
                                            #'keep ZDCDataFramesSorted_*Digis_*_*',
                                            'keep ZDCDataFramesSorted_hcalDigis_*_*',
                                            'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                           'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*',
```

Comment:
this edit is fine, but it is not enough.
edits in customise_PPwithHI.py are needed as well